content/docs/reference/routes/public-access.mdx

```diff
@@ -40,15 +47,15 @@ Enable **Public Access** in the **Policy Builder** in the Console:
 
 | **Name** | **Type** | **Default** | **Usage** |
```

Comment:
for follow-up: the column names are inconsistent. sometimes its "name", sometimes its "annotation", somtimes its "YAML", etc. 

route level options are almost all annotations.

Reply:
Yeah I plan on updating these to be consistent as well. I'll do this in another PR; this one is pretty big already.